optimum/intel/openvino/quantization.py

```diff
@@ -374,10 +374,10 @@ def _quantize_ovbasemodel(
                     quantization_config_copy = copy.deepcopy(quantization_config)
                     quantization_config_copy.dataset = None
                     quantization_config_copy.quant_method = OVQuantizationMethod.DEFAULT
-                    for sd_submodel_name in ["vae_encoder", "vae_decoder", "text_encoder", "text_encoder_2"]:
-                        sd_submodel = getattr(self.model, sd_submodel_name)
-                        if sd_submodel is not None:
-                            _weight_only_quantization(sd_submodel.model, quantization_config_copy)
+                    sub_model_names = ["vae_encoder", "vae_decoder", "text_encoder", "text_encoder_2"]
+                    sub_models = filter(lambda x: x is not None, (getattr(self.model, name) for name in sub_model_names))
```

Comment:
```suggestion
                    sub_models = filter(lambda x: x, (getattr(self.model, name) for name in sub_model_names))
```

Will help with keeping this a single line after `make style`